config/development.json

```diff
@@ -193,24 +192,7 @@
 		"upsell/concierge-session": true,
 		"upsell/nudge-component": true,
 		"woocommerce/extension-dashboard": true,
-		"woocommerce/extension-dashboard-stats-widget": true,
-		"woocommerce/extension-orders": true,
-		"woocommerce/extension-orders-create": true,
-		"woocommerce/extension-orders-edit": true,
-		"woocommerce/extension-products": true,
-		"woocommerce/extension-product-categories": true,
-		"woocommerce/extension-promotions": true,
 		"woocommerce/extension-referrers": true,
```

Comment:
`referrers` is a project that never got finished and remained enabled only in development. A follow-up PR should either remove it, or enable it for all if it's good enough.